### ISSUE TITLE
author issue on new bibliography items

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#568]
Sorry, I can't see if we have raised this before: when you create a new bibliography item it comes up as '[no author]' -- probably until reindexing is done. See bibliography for this MS: https://www.diamm.ac.uk/sources/4950/#/ for an example; items 1 and 3 (though of course by the time you look it may be fixed! Screenshot attached.

<img width="1140" alt="Screenshot 2022-11-18 at 18 07 57" src="https://user-images.githubusercontent.com/10816169/202773454-ff62d6b6-9094-4f53-896c-3fcab52d35a3.png">

**Comment by ahankinson** [comment]
Yes, it's solved by re-indexing (example is fixed now). I'll leave this open though to see if there is a way of solving this without reindexing.

**Comment by juliacmcf** [comment]
Low-low priority

**Comment by fattoxxon** [comment]
Quick getaround: edit the new item & re-save, then author shows up

**Comment by ahankinson** [comment]
Indexing rework will make this less of an issue.
